fast_api_als/database/db_helper.py

```diff
@@ -38,7 +44,19 @@ def insert_lead(self, lead_hash: str, lead_provider: str, response: str):
             'response': response,
             'ttl': datetime.fromtimestamp(int(time.time())) + timedelta(days=constants.LEAD_ITEM_TTL)
         }
-        res = self.table.put_item(Item=item)
+
+        logging.info('Inserting lead...')
+
+        try:
+            res = self.table.put_item(Item=item)
+            try :
+                HTTPStatusCode_Log(res)
+            except Exception as e: 
+                logging.error('HTTPStatusCode could not be logged.' , exc_info = True)
```

Comment:
HTTPStatusCode_Log(res) will never going to throw any exception.

Reply:
I was considering the case when res['ResponseMetadata']['HTTPStatusCode'] inside HTTPStatusCode_Log() doesn't exist due to some unknown reasons.